### PR TITLE
:sparkles: Allow send multiple team invitations at once

### PR DESCRIPTION
**Pull request by pabloalba**
Related to https://tree.taiga.io/project/penpot/us/2798

### CODE REVIEW

File: frontend/resources/styles/main/partials/dashboard-team.scss [hunk 1]
```diff
@@ -19,12 +19,30 @@
...
+    font-size: 14px;
     margin-right: 10px;
+
+    input {
+      padding: 0;
+
+      &.empty {
+        margin-top: 10px;
+        &::placeholder {
+          color: #b1b2b5;
```
Comment (EvaMarco) [inline frontend/resources/styles/main/partials/dashboard-team.scss]: Try to use color variables. They are define in this file frontend/resources/styles/common/dependencies/colors.scss
That color your using here would be `color: $color-gray-20;`

File: backend/src/app/rpc/mutations/teams.clj [hunk 2]
```diff
@@ -330,12 +330,12 @@
...
 
-(s/def ::email ::us/email)
+(s/def ::emails ::us/set-of-emails)
 (s/def ::invite-team-member
-  (s/keys :req-un [::profile-id ::team-id ::email ::role]))
+  (s/keys :req-un [::profile-id ::team-id ::emails ::role]))
 
 (sv/defmethod ::invite-team-member
-  [{:keys [pool] :as cfg} {:keys [profile-id team-id email role] :as params}]
+  [{:keys [pool] :as cfg} {:keys [profile-id team-id emails role] :as params}]
```
Comment (niwinz) [inline backend/src/app/rpc/mutations/teams.clj]: we should keep the both props or reuse it conditionally (i mean email can be single item or set of items).
This is for backward compatibility. We can put a TODO note to remove the "backward compatibility code" for the next release where are way to have incompatible version of frontend running.

File: frontend/src/app/main/ui/components/forms.cljs [hunk 3]
```diff
@@ -74,7 +75,13 @@
                            "password"))))
 
         on-focus  #(reset! focus? true)
-        on-change (fm/on-input-change form input-name trim)
+        on-change (fn [event]
+                    (let [target (dom/get-target event)
+                          value  (if (or (= (.-type target) "checkbox")
+                                         (= (.-type target) "radio"))
+                                   (.-checked target)
```
Comment (niwinz) [inline frontend/src/app/main/ui/components/forms.cljs]: Maybe abstract it to the `app.util.dom` ns? With apropriate `^js` type hint.

File: backend/src/app/rpc/mutations/teams.clj (hunk 2 above)
Comment (pabloalba) [reply backend/src/app/rpc/mutations/teams.clj]: fixed

File: frontend/src/app/main/ui/components/forms.cljs (hunk 3 above)
Comment (pabloalba) [reply frontend/src/app/main/ui/components/forms.cljs]: fixed

File: frontend/resources/styles/main/partials/dashboard-team.scss (hunk 1 above)
Comment (pabloalba) [reply frontend/resources/styles/main/partials/dashboard-team.scss]: fixed

**Pull request merged**